frontend/src/app/page-not-found/page-not-found.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="not-found">
+  <h1>404 Hakemaasi sivua ei löydy. Tarkista sivun osoite.</h1>
```

Comment:
**thought:** It might be a good idea to take the user to home page instead of showing an error page and show a toast to them

Reply:
I added a redirect to the home page instead of this.